STL_Extension/include/CGAL/Named_function_parameters.h

```diff
@@ -360,8 +367,23 @@ bool is_default_parameter(const T&)
   return false;
 }
 
+template <class NP, class TAG>
+constexpr bool is_default_parameter_static()
```

Comment:
why not make the base "is_default_parameter" static directly? (It's not documented.)

Reply:
Same reason as above, you need to change the API because `np` cannot be taken as parameter. I can update it everywhere to use the new API. Should be easy as it is not used much.

Reply:
I meant update everywhere to use the static version, yes.